src/main/java/seedu/recipe/ui/MainWindow.java

```diff
@@ -63,13 +68,25 @@ public MainWindow(Stage primaryStage, Logic logic) {
 
         setAccelerators();
 
+        getRoot().addEventFilter(DeleteRecipeEvent.DELETE_RECIPE_EVENT_TYPE, this::handleDeleteRecipeEvent);
+
+        getRoot().addEventFilter(EditRecipeEvent.EDIT_RECIPE_EVENT_TYPE, this::handleEditRecipeEvent);
```

Comment:
This event handler should be within the popup window, unless you've fixed that

Reply:
Next PR. I'm still doing it on local, but not likely to be ready by 2pm today